packages/core/src/stepFunctions/activation.ts

```diff
@@ -124,6 +125,19 @@ export function initalizeWebviewPaths(
     }
 }
 
+async function startLspServer(extensionContext: vscode.ExtensionContext) {
+    const perflog = new PerfLog('stepFunctions: start LSP client/server')
+    await ASLLanguageClient.create(extensionContext)
+    perflog.done()
+}
+
+function editingASLFromPrevSession() {
+    const aslTextEditors = vscode.window.visibleTextEditors.filter(
+        textEditor => textEditor?.document && ASL_FORMATS.includes(textEditor.document.languageId)
+    )
+    return aslTextEditors.length > 0
```

Comment:
can use `some()` instead of `filter().length > 0`.

the linter will probably flag this if you rebase

Reply:
Updated, thanks!